packages/reactive-core/test/observable.test.ts

```diff
@@ -17,6 +17,19 @@ describe("reactive", () => {
     expect(isReactive(obj)).toEqual(false);
   });
 
+  it("should track changes of a property of an array element", () => {
+    const obj = [
+      { id: 0, val: "a" },
+      { id: 1, val: "b" },
+    ];
+    const obs = reactive(obj);
+    const deriv = obs.find((item) => item.id === 1).val;
+    expect(deriv).toEqual("b");
+
+    obj[1].val = "c";
+    expect(deriv).toEqual("c");
```

Comment:
Hi @jasonm,

This can never work in javascript, because `deriv` is a const. It's value (which is what you check with `toEqual`), can never change.

The other test you've written (where you reassign a value to `dummy` that's triggered by `autorun` works as expected